guides/micronaut-flyway/metadata.json

```diff
@@ -0,0 +1,17 @@
+{
+      "asciidoctor": "micronaut-flyway.adoc",
+      "slug": "micronaut-flyway",
+      "title": "Schema Migration with Flyway",
+      "intro": "Learn how to use the Flyway to manage your schema migrations.",
```

Comment:
remove "the" from in front of Flyway

Learn how to use Flyway to manage your schema migrations.